src/lazy/expanded/macro_table.rs

```diff
@@ -369,38 +373,55 @@ pub enum MacroKind {
 /// A Macro reference that is guaranteed to be valid for its lifespan.
 #[derive(Debug, Copy, Clone, PartialEq)]
 pub struct MacroRef<'top> {
-    id: MacroIdRef<'top>,
-    reference: &'top MacroDef,
+    qualified_address: QualifiedAddress,
+    def: &'top MacroDef,
 }
```

Comment:
🪧 As mentioned prior, `MacroRef` now tracks which module the macro was found in.